src/component/contents/DraftEditorContents.react.js

```diff
@@ -197,21 +199,80 @@ class DraftEditorContents extends React.Component {
         wrapperTemplate,
         key,
         offsetKey,
+        draftWrapperTemplate: configForType.wrapper,
```

Comment:
Can we comment on what the differences between `wrapperTemplate`, `draftWrapperTemplate`, and `customWrapperTemplate` are?

Reply:
Of course, will update shortly.

Reply:
I settled on using `wrapperTemplate` (internal DraftJS impl) and `customWrapperTemplate` (user provided custom wrapper component)